README.md

```diff
@@ -80,6 +80,7 @@ Requirements:
 
 -   Go 1.6 or higher
 -   A [working golang](https://golang.org/doc/code.html) environment
+-   [Protobuf 3.x or higher] (https://developers.google.com/protocol-buffers/docs/downloads)
```

Comment:
I believe you only need protobuf to regenerate protocol buffers files using `make generate`. Changes that don't touch these files won't need protobuf. Maybe add a note to that effect?